.github/workflows/prerelease.yml

```diff
@@ -304,6 +304,59 @@ jobs:
         with:
           name: sv-comp-results
           path: ./sv-comp-results.md
+
+#
+#     Linux kernel benchmark
+#
+  run_linux_kernel_bench:
+    name: "Run Linux kernel test suite"
+    needs: build
+    strategy:
+      matrix:
+        llvm-version: [18]
+        image-version: [22.04]
+        vast-target: ['hl']
+        disable-unsup: [true, false]
+
+    runs-on: ubuntu-${{ matrix.image-version }}
+    timeout-minutes: 360
+    container:
+      image:
+        ghcr.io/trailofbits/vast-ubuntu-${{ matrix.image-version }}-llvm-${{ matrix.llvm-version }}-dev:latest
+
+    steps:
+     - name: Fetch VAST artifact
+       uses: actions/download-artifact@v4
+       with:
+         name: VAST
+
+     - name: Unpack VAST
+       run: mkdir vast && tar -xf VAST-* -C vast --strip-components=1
+
+     - name: Export vast binaries
+       run: echo "${PWD}/vast/bin/" >> $GITHUB_PATH
+
+     - name: Clone the Linux kernel and build it to create compiliation database
+       run: >
+         git clone --depth=1 https://github.com/torvalds/linux.git linux
+         && cd linux
+         && make defconfig
+         && bear -- make LLVM=1
+
+     - name: Download the benchmark runner
+       run: wget https://raw.githubusercontent.com/PappasBrent/vast-benchmarks/add-linux-kernel-benchmark/benchmarks/linux_kernel/run_vast_benchmark.py -O run_vast_benchmark.py
+
+     - name: Run benchmarks
+       run: >
+         python3 run_vast_benchmark.py
+         vast-front # TODO: Determine if this is the correct path to vast-front
+         linux/compile_commands.json
+         vast_linux_kernel_mlir_with_unsup/
+         --num_processes=8
+         --vast_option="-xc"
+         --vast_option="-vast-emit-mlir=hl"
```

Comment:
Take the mlir target from the `vast-target` job variable, so that we can configure it to run into other dialects.